### DISCUSSION TITLE
Trying to understand searches with dates as a parameter

### DISCUSSION THREAD

**Discussion by TigBuh99** [TriliumNext/Trilium#4065] [Q&A] [answered]
I started trying out Trilium and I'm initially examining if it suits some of my specific requirements.

One thing that puzzles me is how to do searches. Using the Task Manager in the Demo files, I wanted to query tasks with future to do dates between say, July 1 2023 and August 31 2023. I couldn't understand how to make such a search/query. How may this be done?

**Comment by zadam** [comment, marked as answer]
Dates follow YYYY-MM-DD format. Something like `#startDate >= '2023-06-30' AND #startDate < '2023-12-31'` should work.

**Reply by TigBuh99** [reply]
Thank you! That works perfectly.
